pandas/io/tests/parser/common.py

```diff
@@ -606,6 +606,21 @@ def test_multi_index_no_level_names(self):
         expected = self.read_csv(StringIO(data), index_col=[1, 0])
         tm.assert_frame_equal(df, expected, check_names=False)
 
+    def test_multi_index_blank_df(self):
+        # GH 14545
+        data = """a,b
+"""
+        df = self.read_csv(StringIO(data), header=[0])
+        expected = DataFrame(columns=[('a',), ('b',)])
```

Comment:
I would expect these to resolve to an ``Index`` actually (we don't have a single level MI)

```
In [6]: pd.MultiIndex.from_tuples([('a',), ('b',)])
Out[6]: Index(['a', 'b'], dtype='object')
```